app/bundles/LeadBundle/Controller/Api/LeadApiController.php

```diff
@@ -607,12 +612,15 @@ protected function preSaveEntity(&$entity, $form, $parameters, $action = 'edit')
 
                 $reason = (int) ArrayHelper::getValue('reason', $dnc, DoNotContact::MANUAL);
 
+                /** @var \Mautic\LeadBundle\Model\DoNotContact $doNotContact */
```

Comment:
```suggestion
                /** @var DoNotContact $doNotContact */
```
Plus lines 414 and 432